pkg/host/internal/network/network.go

```diff
@@ -172,10 +173,19 @@ func (n *network) SetNetdevMTU(pciAddr string, mtu int) error {
 
 func (n *network) GetNetDevMac(ifaceName string) string {
 	log.Log.V(2).Info("GetNetDevMac(): get Mac", "device", ifaceName)
-	macFilePath := filepath.Join(vars.FilesystemRoot, consts.SysClassNet, ifaceName, "address")
-	data, err := os.ReadFile(macFilePath)
+	return readNetDevInterfaceAddress(ifaceName)
+}
+
+func (n *network) GetNetDevGUID(ifaceName string) string {
```

Comment:
what are you getting for ipoib interface ? i think you are getting GID (128bit) and not the node GUID (64 bit)

i think we should fetch GUID via netlink: https://github.com/vishvananda/netlink/blob/06219cde3e811f757f263ae54e848891c8ce6cbc/rdma_link_linux.go#L18

Reply:
You are right:
```
# cat /sys/class/net/ibp75s0f0v1/address
00:00:01:e8:fe:80:00:00:00:00:00:00:00:00:00:00:00:00:00:00
```

Will adjust the code.